Documentation/networking.md

```diff
@@ -1,14 +1,31 @@
+# Networking
+
+`rkt` has two networking options: "host" networking where the apps runs in the root host network namespace or "private" where the apps are allocated a new IP address and a new network namespace.
+
 ## Host (default) mode
 
-By default, `rkt run` will start the pod with host networking. This means that the apps within the pod will share the network stack and the interfaces with the host machine. Since metadata service uses the container IP for indentity, rkt will _not_ register the container with the metadata service in this mode.
+By default, `rkt run` will start the pod with host networking.
+This means that the apps within the pod will share the network stack and the interfaces with the host machine.
+Since metadata service uses the container IP for indentity, rkt will _not_ register the container with the metadata service in this mode.
```

Comment:
s/metadata service/the metadata service/
s/container/pod for consistency (spec will be changed real soon now!)
s/indentity/identity/
maybe add something like "Note that this means that Rocket does not strictly implement the app container specification in this mode" ?